Sources/SwiftDocC/SwiftDocC.docc/Resources/Index.spec.json

```diff
@@ -0,0 +1,119 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "description": "Specification of the Swift-DocC Index.json file.",
+        "version": "0.1.0",
+        "title": "Index"
+    },
+    "paths": {},
+    "components": {
+        "schemas": {
+            "Index": {
+                "type": "object",
+                "required": [
+                    "schemaVersion",
+                    "languages"
+                ],
+                "properties": {
+                    "identifier": {
+                        "$ref": "#/components/schemas/SchemaVersion"
+                    },
+                    "languages": {
+                        "type": "object",
+                        "additionalProperties": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/components/schemas/Node"
+                            }
+                        }
+                    }
+                }
+            },
+            "Node": {
+                "type": "object",
+                "required": [
+                    "title"
+                ],
+                "properties": {
+                    "title": {
+                        "type": "string"
+                    },
+                    "type": {
+                        "type": "string",
+                        "enum": [
+                            "article",
+                            "associatedtype",
+                            "buildSetting",
+                            "class",
+                            "container",
+                            "dictionarySymbol",
+                            "enum",
+                            "enum.case",
+                            "extension",
+                            "func",
+                            "func.op",
+                            "groupMarker",
+                            "httpRequest",
+                            "init",
+                            "languageGroup",
+                            "learn",
+                            "macro",
+                            "method",
+                            "module",
+                            "overview",
+                            "project",
+                            "property",
+                            "propertyListKey",
+                            "propertyListKeyReference",
+                            "protocol",
+                            "resources",
+                            "root",
+                            "sampleCode",
+                            "section",
+                            "struct",
+                            "subscript",
+                            "symbol",
+                            "type.method",
```

Comment:
I have a `tutorial` listed in my code as well. Should I remove it?

Reply:
Currently we use the `"project"` to describe tutorials in both RenderJSON and this new Index spec. It can't hurt to map both `"project"` and `"tutorial"` to the same thing- but I wouldn't expect it to be emitted so you shouldn't need to support it.